src/ipc-handlers.ts

```diff
@@ -1081,6 +1070,95 @@ END` );
 	}
 }
 
+export async function openAppAtPath(
+	_event: IpcMainInvokeEvent,
```

Comment:
@fredrikekelund I am curious, do we need `_event` in this case?

Reply:
All IPC functions always receive an `IpcMainInvokeEvent` object as the first parameter, so we need to include it. However, we don't need to prefix the parameter name with an underscore.

Reply:
I wanted to prefix the parameter with `_` in this case to signal that it's not being used/referenced within the function. But I now realized that I ended using it when falling back to `openURL`, so I updated this in 06bc7ac0.